cli/cmd/cli_unix.go

```diff
@@ -32,7 +36,12 @@ var promptIconsFunc = func(icons *survey.IconSet) {
 // UpdateCommand returns the command that a user should run to update the cli
 // to the latest available version (unix specific command)
 func (c *cliState) UpdateCommand() string {
+	if os.Getenv("LW_HOMEBREW_INSTALL") != "" {
+		return `
+	 $ brew upgrade lacework-cli
+	`
+	}
 	return `
-  $ curl https://raw.githubusercontent.com/lacework/go-sdk/master/cli/install.sh | bash
-`
+	   $ curl https://raw.githubusercontent.com/lacework/go-sdk/master/cli/install.sh | bash
+	 `
```

Comment:
Since this is a string that will be displayed to the user, we can't add tabs or any indentation, lets revert it to:
```suggestion
  $ curl https://raw.githubusercontent.com/lacework/go-sdk/master/cli/install.sh | bash
`
```